src/test/java/Homework17A.java

```diff
@@ -0,0 +1,43 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework17A extends MyBaseTest{
+//    public static WebDriver driver = new ChromeDriver();
+//    we can remove this webdriver initialization
+
+
+    @Test
+    public void addSongtoPlaylist() throws InterruptedException {
+        //login("simplyd2d@gmail.com","te$t$tudent");
+    enterEmail("simplyd2d@gmail.com");
+    enterPassword("te$t$tudent");
+    clickSubmit();
+    Thread.sleep(2000);
+
+    searchSong("Take My Hand");
+    Thread.sleep(2000);
```

Comment:
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.